src/guide/misc-using-binaries.md

```diff
@@ -0,0 +1,173 @@
+- [`kagami`](https://github.com/hyperledger/iroha/tree/iroha2-dev/tools/kagami) — a tool that is used to generate and validate various types of data (e.g., cryptographic keys, genesis blocks, default client and peer configuration files).
+  > See also:
+  > - [Generating Cryptographic Keys](guide/security/generating-cryptographic-keys.md) — instructions on how to generate cryptographic keys with `kagami`.
+  > - [Configure Iroha > Genesis Block: Generation](guide/configure/genesis.md#generation) — instructions on how to generate a default genesis block with `kagami`.
+  > - [Configure Iroha > Peer Configuration: Generation](guide/configure/peer-configuration.md#generation) — instructions on how to generate a default peer configuration file with `kagami`.
+  > - [Configure Iroha > Client Configuration: Generation](guide/configure/client-configuration.md#generation) — instructions on how to generate a default client configuration file with `kagami`.
+
+There are two main ways to work with Iroha 2 binaries:
+
+1. [Using the pre-built Docker images](#docker-install).
+2. [Using the source GitHub repository](#source):
+    - [Installing from source GitHub repository](#source-install).
+    - [Running from cloned GitHub repository](#source-run).
+
+## Using the Pre-Built Docker Images {#docker-install}
+
+First, install the following Docker prerequisites:
+
+- [Docker](https://docs.docker.com/get-docker/)
+- [Docker Compose](https://docs.docker.com/compose/install/)
+
+To install a binary from a pre-built Docker image, execute one of the following commands:
+
+::: code-group
+
+```shell [iroha]
+docker run -t hyperledger/iroha2:dev
+```
+
+```shell [iroha_client_cli]
+docker run -t hyperledger/iroha2:client-cli-dev
+```
+
+```shell [kagami]
+docker run -t hyperledger/iroha2:kagami-dev
+```
+
+:::
+
+::: info
+
+Listed examples only reference the `dev` versions of the Docker images.
+
+For a list of all available Docker images for Iroha 2, see the [Iroha 2 Docker Hub](https://hub.docker.com/r/hyperledger/iroha2) page.
+
+:::
+
+## Using the Source GitHub Repository {#source}
+
+To perform any of the actions with the GitHub repository (i.e., [building & installing](#source-install), or [running](#source-run) binaries), the [Rust toolchain](https://www.rust-lang.org/) must first be installed. The toolchain is delivered with the package manager—[Cargo](https://doc.rust-lang.org/cargo/index.html)—that allows for installation of Rust crates from different sources, including GitHub.
+
+The recommended way to install the Rust toolchain is through a Rust installer and version management tool—[Rustup](https://rust-lang.github.io/rustup/) (See also: [Install Rust](https://www.rust-lang.org/tools/install))—by executing the following command in your terminal:
```

Comment:
don't we already have the instructions about rust/rustup in the installation guide?